docs/architecture/security-and-data-sovereignty.md

```diff
@@ -0,0 +1,124 @@
+
+- **Access Tokens**: Short-lived JWT tokens facilitating immediate Kafka topic access.
+
+### 2.2 Token Lifecycle
+
+Scenarios managed by token lifecycle processes:
+
+- **Normal Expiration**: Automated token refresh prior to expiration.
+- **Forced Revocation**: Immediate token revocation capability.
+- **Failed Refresh**: Consumer access terminated upon token refresh failure.
+
+### 2.3 Asset Role
+
+- The **Kafka topic** is the negotiated asset, not individual records.
+- Topic-level negotiation and access control.
+- Consumers access all messages published during token validity.
+
+## 3. Security Measures
+
+### 3.1 Security Protocols
+
+Kafka Extension security protocols include:
+
+- **Transport Encryption**: Secure communications using TLS/SSL (SASL_SSL protocol).
+- **Authentication**: OAuth 2.0 tokens via SASL/OAUTHBEARER.
+- **Authorization**: Comprehensive authorization via EDC policies, Kafka ACLs, and OAuth scopes.
+
+### 3.2 Protection Against Unauthorized Access
+
+Security features preventing unauthorized access:
+
+- **Dynamic Credentials**: Temporary, single-use credentials.
+- **Short-lived Tokens**: Reducing potential misuse windows.
+- **Secure Storage**: Credentials managed securely in Vault.
+- **Token Validation**: OAuth Service token validation prior to Kafka access.
+
+## 4. Self-Sovereign Data Sharing
+
+### 4.1 Autonomous Control
+
+Providers maintain full autonomous control over data sharing:
+
+- **Policy Definition**: Customizable data access policies.
+- **Dynamic Management**: Real-time policy enforcement via Control Plane.
+- **Data Residency**: Data remains on provider infrastructure, access-only model.
+- **Contractual Governance**: Data access governed by explicitly negotiated contracts.
+
+### 4.2 Granular Access Control
+
+Fine-grained access capabilities provided by the Kafka Extension include:
+
+- Controlled visibility of contract offers.
+- Varied policy applications to assets.
+- Topic-policy mapping with precise constraints.
+
+### 4.3 Data Sovereignty Scenarios
+
+Illustrative scenarios demonstrating data sovereignty:
+
+1. **Time-Limited Access:**  
+   Defined duration access, automatically terminated post-expiration.
+
+2. **Purpose-Specific Access:**  
+   Explicit use-case limitations such as quality assurance only.
+
+3. **Selective Revocation:**  
+   Immediate revocation for misuse incidents without wider service impact.
+
+4. **Auditability:**  
+   Complete records of negotiation and data transfers for compliance verification.
+
+5. **Differentiated Access Levels:**  
+   Persistent policies for long-term customers. Short-term revocable policies for consultants and auditors.
+
+## Conclusion
+
+The Tractus-X EDC Kafka Extension integrates robust security measures and comprehensive data sovereignty controls, enabling data providers to securely manage and share real-time data while maintaining full governance over their assets.
+
+## Glossary
+
+- **EDC**: Eclipse Dataspace Connector
```

Comment:
Add 
* Control Plane 
* Data Plane 
* Vault
* OAuth Scopes
* Kafka Topic
 to glossary and short description